docs/source/release-notes/4.0.0.rst

```diff
@@ -0,0 +1,48 @@
+4.0.0: 2023-04-18
+-----------------
+
+Backwards Incompatible Changes
+``````````````````````````````
+
+- repo.compare_commits doesn't use pagination (@amaccormack-lumira).
+  See also gh-1137_
```

Comment:
There are older notes (I think from 1.0 maybe?) Where we've renamed attributes. Can you look at those and make sure this calls out the attribute rename?